src/Components/NavBar/Menus/DropDownSection.tsx

```diff
@@ -0,0 +1,88 @@
+import { Box, color, Flex, MenuItem, Sans } from "@artsy/palette"
+import React from "react"
+import { MenuLinkData, SimpleLinkData } from "../menuData"
+
+interface DropDownSectionProps {
+  section: MenuLinkData
+}
+
+export const DropDownSection: React.FC<DropDownSectionProps> = ({
+  section,
+}) => {
+  if (!section) return null
+
+  return (
+    <>
+      {section.text === "Artist Nationality & Region" ? (
+        <Flex>
+          <Box width={[110, 110, 110, 135, 150]} py={4} mr={[2, 2, 3, 3]}>
+            <Sans size="2" mb={1}>
+              {section.text}
+            </Sans>
+            {section.menu &&
+              section.menu.links.slice(0, 8).map((menuItem: SimpleLinkData) => {
```

Comment:
Instead of explicitly breaking this into two separate columns, could we leverage flexbox to generate the layout based on our list of items? I know it's hard-coded data right now, but I think this'll be easier to maintain if the layout can adjust based on the number of items.

Reply:
<img width="465" alt="Screen Shot 2020-04-14 at 12 39 04 PM" src="https://user-images.githubusercontent.com/5201004/79269169-02649200-7e6a-11ea-81ba-347139bacdf9.png">

@sweir27 I thought about that and explored it for a bit. All of the `MenuItem` links fit in a container with a specified width. What makes this UI somewhat tricky is that ['Artists Nationality & Region'](https://github.com/artsy/reaction/blob/7d9348252dce9d66dc30a820c2b86143547115aa/src/Components/NavBar/menuData.ts#L302-L367) is one long array of links that needs to break off (halfway) and fit in a new container with the specified width. Do you maybe have some time to chat about the implementation/pair?